Parsers/CiscoMeraki/MerakiSecurityEvents.txt

```diff
@@ -0,0 +1,20 @@
+// Title:           Cisco Meraki SecurityEvents API
+// Author:          Netrix LLC - Rich Lilly
+// Version:         1.0
+// Last Updated:    1/18/2021
+// Comment:         Initial Release
+//  
+// DESCRIPTION:
+// This parser takes raw Cisco Meraki logs from the securityEvents Cloud API and parses the logs into a normalized schema.
+//
+// USAGE:
+// 1. Open Log Analytics/Azure Sentinel Logs blade. Copy the query below and paste into the Logs query window. 
+// 2. Click the Save button above the query. A pane will appear on the right, select "as Function" from the drop down. Enter a Function Name.
+//    It is recommended to name the Function Alias, as MerakiSecurityEvents
+// 3. Kusto Functions can typically take up to 15 minutes to activate. You can then use Function Alias for other queries.
+//
+// REFERENCES: 
```

Comment:
Please include inline Log Sample event so others understand the type of data that should be included.  See the other Meraki parser as an example - https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/CiscoMeraki/CiscoMeraki.txt